src/avt/MFEM/avtMFEMDataAdaptor.C

```diff
@@ -403,6 +414,16 @@ avtMFEMDataAdaptor::RefineMeshToVTK(mfem::Mesh *mesh,
 
     AVT_MFEM_INFO("High Order Mesh is not periodic.");
 
+    // mfem::Mesh::MakeRefined does not yet support pyramids
+    if(mesh->GetNE() > 0)
+    {
+        if(mesh->GetElement(0)->GetType() == mfem::Element::PYRAMID)
+        {
+            AVT_MFEM_EXCEPTION1(InvalidVariableException,
+                                "Current MFEM implementation does not support refining Meshes with Pyramids.");
+        }
+    }
+
```

Comment:
Does legacy refine handle the pyramid case?

Reply:
Doesn't crash, but result isn't right.

![Screenshot 2024-10-30 at 8 32 46 AM](https://github.com/user-attachments/assets/e84c6494-a2d0-4500-a4aa-3efce7a4fd42)